aksel.nav.no/website/components/website-modules/OutdatedAlert.tsx

```diff
@@ -20,9 +20,13 @@ export default function OutdatedAlert() {
         </BodyLong>
         {ctx?.validUser && (
           <>
+            <BodyLong className="mb-2 mt-2">
+              <Link href="https://aksel.nav.no/side/skriv-for-aksel#a5b79ddd59da">
+                Hvordan oppdaterer man innhold i Sanity?
+              </Link>
+            </BodyLong>
             <BodyLong className="mb-2">
-              Du ser også dette fordi du kan redigere denne artikkelen. Har du
```

Comment:
I felt the "Du ser dette.." to be a bit noisy / redundant as well, derailing thought to "også?", it refers to conditional logic inside the component that is only present when you are the article owner, but I think it's more than fine to simply "talk directly to the owner as an owner" in this case 🤔